rust/integration-tests/src/cli_parser.rs

```diff
@@ -0,0 +1,52 @@
+use once_cell::sync::Lazy;
+use std::sync::Mutex;
+
+#[derive(Debug, Clone)]
+pub struct TestArgs {
+    pub generate_output: bool,
+    pub output_path: Option<String>,
+}
+
+// Define a global static to store the parsed arguments
+#[allow(dead_code)]
+static TEST_CONFIG: Lazy<Mutex<TestArgs>> = Lazy::new(|| {
+    let args = parse_test_args();
+    Mutex::new(args)
+});
+
+// Function to fetch global test args
+#[allow(dead_code)]
+pub fn get_test_config() -> (bool, Option<String>) {
+    let test_args = TEST_CONFIG.lock().unwrap().clone();
+    (test_args.generate_output, test_args.output_path)
+}
+
+pub fn parse_test_args() -> TestArgs {
```

Comment:
Could we also reuse the one defined in the SDK?

Reply:
which function are you referring to?

Reply:
Oh I thought parse_test_args is part of the SDK. I feel like it should be part of the testing framework SDK, wdyt?

Reply:
yeah also thought about that whether we should let them decide to use their own way of passing those args (generate-files and custom_path), b/c they might want more customzied way. but we can expose first and gather their feedback

Reply:
https://github.com/aptos-labs/aptos-indexer-processor-sdk/pull/72